app/components/work_video_show_component.rb

```diff
@@ -32,9 +32,9 @@ def video_src_url
   # the representative, if it's visible to current user, otherwise nil!
   def video_asset
     return @video_asset if defined?(@video_asset)
-
+    show_all_members = (can? :read, Asset) && (can? :read, Work)
```

Comment:
Yeah, as you repeat this `show_all_members` thing over and over... I'm leaning toward, yeah, let's make an ApplciationHelper helper method for this, and document it?  (and maybe the definition is just `can? :read, Kithe::Model` if that works.

Reply:
OR a method on `User`?  `current_user.can_see_unpublished_records?` ?

Reply:
We did create an ApplicationHelper helper method, which in turn looks at a method in the policy class.